dmutils/flask_init.py

```diff
@@ -56,6 +56,7 @@ def add_header(response):
     application.add_template_filter(formats.dateformat)
     application.add_template_filter(formats.datetimeformat)
     application.add_template_filter(formats.datetodatetimeformat)
+    application.add_template_filter(formats.localizeddatetimeformat)
```

Comment:
Let's add a tech debt ticket to review all of our date time filters. Having six is mildly terrifying 😨 (although having this one feels like a _good_ thing regardless).

Reply:
https://trello.com/c/jyQZCEB7/303-review-datetime-filters